### PR TITLE
Change messages around!

### PR DESCRIPTION
**Pull request by feedmypixel**
## add a message to the Verify win stage
screen shot 2017-11-27 at 11 56 40

## remove the message from the prospect stage
screen shot 2017-11-27 at 11 57 45

### CODE REVIEW

File: src/apps/investment-projects/middleware/shared.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ async function getInvestmentDetails (req, res, next) {
         name: stageName,
         isComplete: investmentData.team_complete && investmentData.requirements_complete && investmentData.value_complete,
         incompleteFields: buildIncompleteFormList(get(investmentData, 'incomplete_fields', [])),
-        messages: get(toCompleteStageMessages, lowerCase(stageName), []),
+        messages: get(toCompleteStageMessages, camelCase(stageName), []),
```
Comment (teneightfive) [inline src/apps/investment-projects/middleware/shared.js]: Do you want `camelCase`? Won't this strip spaces if it's multiple words?

Comment (ztolley) [reply src/apps/investment-projects/middleware/shared.js]: it will

Comment (feedmypixel) [reply src/apps/investment-projects/middleware/shared.js]: Yup this is so it matches the value in the `toCompleteStageMessages` object

Comment (teneightfive) [reply src/apps/investment-projects/middleware/shared.js]: Ah, gotcha! Yep makes sense.

**Pull request merged**